### PR TITLE
Use sensible config thing

### PR DESCRIPTION
**Pull request by zuzak**

### CODE REVIEW

File: modules/news.js [hunk 1]
```diff
@@ -185,9 +185,9 @@ module.exports = {
       }
     },
     news: function (bot, news) {
-      console.log(news)
+      //console.log(news)
```
Comment (danhedron) [inline modules/news.js]: Is this line worth keeping

File: modules/_commands.js [hunk 2]
```diff
@@ -28,8 +29,8 @@ module.exports = {
                 return
               }
               if (cmd.privileged) {
-                if (bot.config.irc.control !== msg.to) {
-                  if (bot.config.irc.insecure === true) {
+                if (bot.config.get('irc.control') !== msg.to) {
```
Comment (danhedron) [inline modules/_commands.js]: Should this be `irc.control !== msg.to || !irc.control_enabled`?

Comment (zuzak) [reply modules/_commands.js]: We check if control channels are a thing in the next line down

Comment (danhedron) [reply modules/_commands.js]: I can't leave mutliline comments :cry: 

For readability and making `insecure` clearly different from the `secure` configuration.

**Pull request merged**